tests/flow/test_tdigest.py

```diff
@@ -170,79 +170,123 @@ def test_negative_tdigest_add(self):
+        self.assertOk(self.cmd("tdigest.create", "from-3", "compression", 300))
         # insert datapoints into sketch
         self.assertOk(self.cmd("tdigest.add", "from-1", 1.0, 1.0))
         self.assertOk(self.cmd("tdigest.add", "from-2", 1.0, 10.0))
-        # merge to a t-digest with default compression
-        self.assertOk(self.cmd("tdigest.mergestore", "to-tdigest-100", "2","from-1", "from-2"))
-        # assert we have same merged weight on both histograms ( given the to-histogram was empty )
+        # merge to a t-digest with max compression of all inputs which is 200
+        self.assertOk(self.cmd("tdigest.merge", "to-tdigest-100", "2", "from-1", "from-2"))
         to_info = parse_tdigest_info(self.cmd("tdigest.info", "to-tdigest-100"))
         # ensure tha the destination t-digest has the largest compression of all input t-digests
         compression = int(to_info["Compression"])
         self.assertEqual(200, compression)
+        # assert we have same merged weight on both histograms ( given the to-histogram was empty )
         total_weight_to = float(to_info["Merged weight"]) + float(
             to_info["Unmerged weight"]
         )
         total_weight_from = 10.0 + 1.0
         self.assertEqual(total_weight_from, total_weight_to)
 
+        # merge to a t-digest that already exists so we will preserve its compression
+        self.assertOk(self.cmd("tdigest.merge", "to-1", "2", "from-1", "from-2"))
+        to_info = parse_tdigest_info(self.cmd("tdigest.info", "to-1"))
+        # ensure tha the destination t-digest has the largest compression of all input t-digests
+        compression = int(to_info["Compression"])
+        self.assertEqual(55, compression)
+
         # merge to a t-digest with non-default compression
-        self.assertOk(self.cmd("tdigest.mergestore", "to-tdigest-50", "2","from-1", "from-2", "COMPRESSION", "50"))
+        self.assertOk(self.cmd("tdigest.merge", "to-tdigest-50", "2","from-1", "from-2", "COMPRESSION", "50"))
         # ensure tha the destination t-digest has the passed compression
         to_info = parse_tdigest_info(self.cmd("tdigest.info", "to-tdigest-50"))
         compression = int(to_info["Compression"])
         self.assertEqual(50, compression)
 
-    def test_tdigest_mergestore_percentile(self):
+        # merge to a t-digest that already exists with non-default compression
+        self.assertOk(self.cmd("tdigest.merge", "to-tdigest-50", "2","from-1", "from-2", "COMPRESSION", "500"))
+        # ensure tha the destination t-digest has the passed compression
+        to_info = parse_tdigest_info(self.cmd("tdigest.info", "to-tdigest-50"))
+        compression = int(to_info["Compression"])
+        self.assertEqual(500, compression)
+
+        # merge to a t-digest that already exists but given we specify override it will use the max compression of all inputs
+        self.assertOk(self.cmd("tdigest.merge", "to-tdigest-50", "2","from-1", "from-2", "OVERRIDE"))
```

Comment:
Can we override the same key or then the result will be the same?
`self.assertOk(self.cmd("tdigest.merge", "from-1, "2","from-1", "from-2", "OVERRIDE"))`

Reply:
we can override. I've added a specific test to illustrate that: 
https://github.com/RedisBloom/RedisBloom/blob/a877bfe9ca3b717a6abba945446945f2bfb4ee23/tests/flow/test_tdigest.py#L193